packages/protobuf-test/extra/example.proto

```diff
@@ -15,8 +15,11 @@
 syntax = "proto3";
 package docs;
 
-message Example {
-  string foo = 1;
-  bool bar = 2;
-  Example baz = 3;
+message User {
```

Comment:
https://github.com/bufbuild/protobuf-es/blob/main/docs/runtime_api.md is using this definition and needs updating if we change it.

Reply:
Fixed.  Hopefully I didn't miss anything.